generator/generator_tool/generator_tool.cpp

```diff
@@ -175,6 +188,21 @@ int main(int argc, char ** argv)
     classif().SortClassificator();
   }
 
+  // Load mwm tree only if we need it
+  std::unique_ptr<storage::Storage> storage = nullptr;
+  if (FLAGS_make_routing_index || FLAGS_make_cross_mwm)
+  {
+    if (!FLAGS_countries_filename.empty())
+    {
+      storage.reset(new storage::Storage(FLAGS_countries_filename, FLAGS_countries_dir));
```

Comment:
Should I use "std/unique_ptr.hpp" instead of <memory> to make make_unique available?